frontend/test/metabase-smoketest/admin_setup.cy.spec.js

```diff
@@ -0,0 +1,977 @@
+      cy.contains(
+        'Once you\'re there, give it a name and click "Add bot integration". Then copy and paste the Bot API Token into the field below. Once you are done, create a "metabase_files" channel in Slack. Metabase needs this to upload graphs.',
+      );
+    });
+
+    it("should create new groups", () => {
+      cy.findByText("People").click();
+
+      cy.findByText("2 other groups");
+
+      cy.findAllByText("Groups")
+        .first()
+        .click();
+
+      cy.findByText(
+        "You can use groups to control your users' access to your data. Put users in groups and then go to the Permissions section to control each group's access. The Administrators and All Users groups are special default groups that can't be removed.",
+        { exact: false },
+      );
+      cy.findByText("All Users");
+      cy.findByText("Slack").should("not.exist");
+
+      // Creates new group
+
+      cy.findByText("Create a group").click();
+      cy.get("input").type("Marketing");
+      cy.findByText("Add").click();
+      cy.findByText("Marketing").click();
+
+      cy.findByText("A group is only as good as its members.");
+
+      // Adds no collection user as member
+
+      cy.findByText("Add members").click();
+      cy.get("input").type("T");
+      cy.findByText(
+        USERS.nocollection.first_name + " " + USERS.nocollection.last_name,
+      ).click();
+      cy.findByText("Add").click();
+
+      cy.findByText(USERS.nocollection.username);
+      cy.findByText("A group is only as good as its members.").should(
+        "not.exist",
+      );
+
+      // Adds self as member
+
+      cy.findByText("Add members").click();
+      cy.get("input").type("T");
+      cy.findByText(
+        USERS.admin.first_name + " " + USERS.admin.last_name,
+      ).click();
+      cy.findByText("Add").click();
+
+      cy.findByText(USERS.admin.username);
+
+      // Check member count
+
+      // *** Unnecessary click (Issue #12693)
+      cy.findAllByText("People")
+        .last()
+        .click();
+
+      cy.findAllByText("2 other groups").should("have.length", 2);
+
+      cy.findAllByText("Groups")
+        .first()
+        .click();
+
+      cy.findByText("Marketing");
+      cy.get("td")
+        .eq("-2")
+        .contains("2");
+    });
+
+    it("should create new users in different groups", () => {
+      cy.findAllByText("People")
+        .last()
+        .click();
+
+      // Add new user into 2 groups
+
+      cy.findByText("Add someone").click();
+      cy.findByLabelText("First name").type(new_user.first_name);
+      cy.findByLabelText("Last name").type(new_user.last_name);
+      cy.findByLabelText("Email").type(new_user.username);
+      cy.get(".ModalBody")
+        .find(".Icon-chevrondown")
+        .first()
+        .click();
+      cy.findByText("English").click();
+      cy.findByText("Turkish").should("not.exist");
+      cy.wait(1000)
+        .findAllByText("Default")
```

Comment:
```suggestion
      cy.findAllByText("Default")
```

I'm not sure if something changed from when we first started writing these tests, but selecting the language when creating a user doesn't seem to be an option anymore.

Reply:
Oh, how interesting